root.sh

```
@@ -55,7 +55,7 @@ incremental_recipe: |
   make ${JOBS:+-j$JOBS} install
   mkdir -p $INSTALLROOT/etc/modulefiles
   rsync -a --delete etc/modulefiles/ $INSTALLROOT/etc/modulefiles
-  cd $INSTALLROOT/test
+  cd $INSTALLROOT/test || exit
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)